ToDo/src/css/todo/index.css

```diff
@@ -0,0 +1,26 @@
+/* GENERAL STYLES */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(195, 142, 166);
```

Comment:
it's better to avoid applying specific style to body, move this style to todo-wrapper.

Reply:
changed